packages/lint-worker/src/helpers.ts

```diff
@@ -0,0 +1,50 @@
+import { compareMajorMinorVersions } from './version';
+import {
+  DbSchema as DbSchema2,
+  Neo4jFunction,
+  Neo4jProcedure,
+} from '@neo4j-cypher/language-support';
+import { Neo4jSchemaPoller } from '@neo4j-cypher/query-tools';
+import { DbSchema as DbSchema1 } from 'languageSupport-next.13';
+
+export function convertDbSchema(
+  originalSchema: DbSchema2,
+  neo4j: Neo4jSchemaPoller,
+): DbSchema2 | DbSchema1 {
```

Comment:
I'd prefer this to be named `DbSchemaV2` and `DbSchemaV1` as I asked in the other PR so it's clear it's two different versions of the schema